src/content/self-hosted/manage/buildkit-high-performance.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Optimizing Build Performance
+description: Best Practices for Optimal Build Performance
+id: buildkit-high-performance
```

Comment:
this slug assumes that people now what buildkit it. I think we should rename it to something closer to what the user's are looking for (maybe call it 'build service' or 'fine tune build service' ?)

Reply:
I am using BuildKit on purpose because this is more than image builds

Reply:
Buildkit is the name of the open source project, not of our features/product. I think we should stay to okteto terms to help people find the issues in the docs, and understand how everything fits together (especially now that the build service is more than just builds)

Reply:
`buildkit` is the name in our Helm configuration file. It's also the name of the statefulset. Since this document talks about the `buildkit` setting and the `buildkit` statefulset, I think using `BuildKit` is more appropriate and also coverts "remote execution"